configure.ac

```diff
@@ -1673,11 +1678,16 @@ fi
 
 dnl Remove the "-g" option if set for GCC.
 if [[ "x$GCC" = "xyes" ]]; then
+  if test "$NEED_GNU_SED" = "yes" ; then
+    SED=gsed
+  else
+    SED=sed
+  fi
   if test "$ac_test_CFLAGS" != set; then
-    CFLAGS=`echo $CFLAGS | sed -e 's/-g\b//g'`
+    CFLAGS=`echo $CFLAGS | ${SED} -e 's/-g\b//g'`
```

Comment:
Hi,

This don't seem to work under osx.
Why not simply use 
`sed "s/-g$//g" | sed "s/-g //g"`

Reply:
Yes, that's a much more practical solution.